frontend/src/libs/components/chat-sidebar/chat-card-item/chat-card-item.tsx

```diff
@@ -0,0 +1,34 @@
+import { useCallback } from '#libs/hooks/hooks.js';
+
+import { Card } from '../../components.js';
+
+type Properties = {
+  chat: Chat;
+};
+
+type Chat = {
+  id: number;
+  name: string;
+};
+
+const mockedSelectedChat = {
+  id: 1,
+};
+
+const ChatCardItem: React.FC<Properties> = ({ chat }) => {
+  const handleSelectChat = useCallback((): void => {
+    mockedSelectedChat.id = chat.id; //change to redux logic
+  }, [chat.id]);
+
+  return (
+    <Card
+      title={chat.name}
+      imageUrl="images/card-image-placeholder.png"
+      onPress={handleSelectChat}
+      isActive={mockedSelectedChat.id === chat.id}
+      key={chat.id}
```

Comment:
You don't need key, you're not looping here

Reply:
done